### ISSUE TITLE
Duplicate symbols for architecture x86_64

### ISSUE THREAD

**Issue by miguelgazela** [kmussel/Moment#15]
I've tried adding the 4 files to an existing project and it doesn't compile. It gives a bunch of 'duplicated symbols' errors for variables contained in the TimeParser file.

Tried it using a completely new project and faced the same problem.

Do I need to change any configuration in the project compiler phase?

Thanks in advance.

![screen shot 2016-03-04 at 10 54 04](https://cloud.githubusercontent.com/assets/812433/13525827/a95001cc-e1fb-11e5-84f2-727c976d44a3.png)

**Comment by kmussel** [comment]
yeah I remember running into that.  There was a configuration change for that.  Will have to check later but until then you can checkout https://github.com/momentlib.  There is a swift framework project that you could compare your configuration with.
